<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>348</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add backup profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Profile name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_save">
     <property name="maximumSize">
      <size>
       <width>101</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="t_profileName"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sourcesTab">
      <attribute name="title">
       <string>Sources</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_addFolders">
         <property name="text">
          <string>Add folders</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_addFiles">
         <property name="text">
          <string>Add files</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_remove">
         <property name="text">
          <string>Remove item</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QListWidget" name="list_Files"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <widget class="QCheckBox" name="cb_compress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Compression</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_enc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Encryption</string>
       </property>
      </widget>
      <widget class="QComboBox" name="list_comp">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>141</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>xz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lzma</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bzip2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gzip</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="list_enc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>141</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>aes-256-cbc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rc4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>des</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>des3</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="bst_setDest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose destination...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="t_dest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>491</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>491</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_compress</sender>
   <signal>toggled(bool)</signal>
   <receiver>list_comp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_enc</sender>
   <signal>toggled(bool)</signal>
   <receiver>list_enc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
